client/src/ui/molecules/document-survey/surveys.ts

```diff
@@ -20,18 +20,32 @@ export interface Survey {
 enum SurveyBucket {
   BROWSER_SURVEY_OCT_2022 = "BROWSER_SURVEY_OCT_2022",
   CSS_CASCADE_2022 = "CSS_CASCADE_2022",
+  FIREFOX_WEB_COMPAT = "FIREFOX_WEB_COMPAT",
   INTEROP_2023 = "INTEROP_2023",
 }
 
 enum SurveyKey {
   BROWSER_SURVEY_OCT_2022 = "BROWSER_SURVEY_OCT_2022",
   CSS_CASCADE_2022_A = "CSS_CASCADE_2022_A",
   CSS_CASCADE_2022_B = "CSS_CASCADE_2022_B",
+  FIREFOX_WEB_COMPAT = "FIREFOX_WEB_COMPAT",
   INTEROP_2023_CSS_HTML = "INTEROP_2023_CSS_HTML",
   INTEROP_2023_API_JS = "INTEROP_2023_API_JS",
 }
 
 export const SURVEYS: Survey[] = [
+  {
+    key: SurveyKey.FIREFOX_WEB_COMPAT,
+    bucket: SurveyBucket.FIREFOX_WEB_COMPAT,
+    show: (doc: Doc) =>
+      /en-US\/docs\/Web\/(JavaScript|API)(\/|$)/i.test(doc.mdn_url),
+    src: "https://survey.alchemer.com/s3/7195211/Which-web-platform-features-should-be-implemented-in-Firefox",
```

Comment:
Hmm can we update the URL to match the new title?

Reply:
I've edited the URL in Alchemer.
```suggestion
    src: "https://survey.alchemer.com/s3/7195211/Help-prioritize-cross-browser-web-platform-features",
```